### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by yokofly**
PR checklist:
- Did you run ClangFormat ?
- Did you separate headers to a different section in existing community code base ?
- Did you surround `proton: starts/ends` for new code in existing community code base ?

Please write user-readable short description of the changes:

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,6 +1,17 @@
-# Contributing to ClickHouse Documentation
 
-## Why Do You Need to Document ClickHouse
+Welcome to the Proton database documentation.
+
+# Please note that the docs directory currently contains [ClickHouse](https://github.com/clickhouse/clickhouse.git)-related content. For Proton-specific documentation, kindly visit:
+
+Proton Documentation Site: [https://docs.timeplus.io/proton](https://docs.timeplus.com/proton)
```
Comment (jovezhong) [inline docs/README.md]: Proton documentation: [https://docs.timeplus.io/proton](https://docs.timeplus.com/proton)

File: docs/README.md [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
+
+GitHub Documentation: [Proton Guide](https://github.com/timeplus-io/docs/blob/main/docs/proton.md)
```
Comment (jovezhong) [inline docs/README.md]: Issues and PR are welcome: https://github.com/timeplus-io/docs

**Review by jovezhong — CHANGES_REQUESTED** [review]
minor changes

**Comment by yokofly** [comment]
we should open some easy tasks to make people familiar this huge  codebase if u expected pr

my thought:
easy tasks shall be no ddl requirement and assume  the outside contributor  can exit at any stage.(no pressure to the contributor and give them proper help/feedback)

**Comment by yokofly** [comment]
open a new incognito window and cannot access https://docs.timeplus.io/proton
I am going to switch to `.com`

**Review by jovezhong — APPROVED** [review]

**Pull request merged**